src/bpftrace.cpp

```diff
@@ -1175,6 +1178,13 @@ int BPFtrace::run(BpfBytecode bytecode)
   if (std::getenv("__BPFTRACE_NOTIFY_PROBES_ATTACHED"))
     std::cerr << "__BPFTRACE_NOTIFY_PROBES_ATTACHED" << std::endl;
 
+#ifdef HAVE_LIBSYSTEMD
+  err = sd_notify(false, "READY=1\nSTATUS=Processing requests...");
+  if (err < 0)
+    LOG(ERROR) << "Failed to send readiness notification, ignoring: "
```

Comment:
If we're not exiting or aborting this should be `LOG(WARNING)` - same goes for the one below.